### DISCUSSION TITLE
Add full "collection experience" to Roles and Presets

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#6937] [Feature Requests]
### Describe the Bug

I don't know if I would call it a bug. But on the Roles & Permissions page, I can't adjust the table to view records that are long and I can't filter(Search) the table too. I plan to create a bunch of roles and we will need to be able to filter(Search) through them. Same with the presets and bookmarks page.

https://user-images.githubusercontent.com/9326922/126758897-a206abd4-694b-48a1-a998-24508467c9a5.mp4

### To Reproduce

Vist the roles and permissions page or the presets and bookmarks page

### What version of Directus are you using?

RC85

### What version of Node.js are you using?

14.15.3

### What database are you using?

10.4.17-MariaDB - Source distribution

### What browser are you using?

Chrome

### What operating system are you using?

MacOS

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
The difficulty here is that the data shown isn't the data as stored in the database, so the regular collection flow doesn't just work 1-1

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
